README.md

```diff
@@ -1,24 +1,78 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Budget App
+> This App is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
-Things you may want to cover:
+## Screenshots
+![Screenshot from 2022-11-19 22-01-14](https://user-images.githubusercontent.com/75738563/202867238-3da627a7-0456-4e31-9d7a-7dd1bea5c883.png)
 
-* Ruby version
+## Built With :hammer_and_wrench:
 
-* System dependencies
+- Major languages: Ruby
+- Frameworks: Ruby on Rails
+- Technologies used: Rubocop, gem, Postgresql
 
-* Configuration
+## Live Demo (if available)
 
-* Database creation
+[Live Demo Link](https:https://gleaming-douhua-695c31.netlify.app/)
```

Comment:
- [ ] the link to your live demo is not yet working as the previous CR suggested.